hooks/post_gen_project.py

```diff
@@ -114,7 +106,7 @@ def remove_copying_files():
 not_oss = '{{ cookiecutter.open_source_license }}' == 'Not open source'
 
 # 1. Generates and saves random secret key
-make_secret_key(PROJECT_DIRECTORY)
+make_secret_key(PROJECT_DIRECTORY, ['CHANGEME!!!', 'something long and random'])
```

Comment:
They could all just be the same sentinel, no? Like, `CHANGEME!!!` or `this should be replaced at project creation time`.